Ruby-Assignment-2-LoopingsAndLiterals/Question-1/solution2.rb

```diff
@@ -0,0 +1,31 @@
+
+TOTAL_CASES = gets.chomp.to_i
+
+# method for calculating sum of array elements
+def sum_of_numbers(numbers)
+  total = numbers.reduce(:+)
```

Comment:
great.. and since i only have only statement, what if I also remove the function sum_of_numbers and use the statement directly in if condition.